ipmiMgrApp/Db/fru_common_alias.template

```diff
@@ -23,3 +23,22 @@ alias("$(dev):$(id)$(unit):PSN", "$(P)$(R)$(id)ProdSN-Cte")
 alias("$(dev):$(id)$(unit):PPARTNUMBER", "$(P)$(R)$(id)ProdPN-Cte")
 
 alias("$(dev):$(id)$(unit):POWERCTL", "$(P)$(R)$(id)PowerCtl-Sel")
+
+record(bo, "$(P)$(R)$(id)SoftReset-Cmd") {
```

Comment:
Considering the naming convention, maybe this should be named `SoftRst-Cmd` instead.